easybuild/easyconfigs/c/c2sm/c2sm-15.08.2018-kesch.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'Bundle'
+
+name = 'c2sm'
+version = '15.08.18'
+
+homepage = 'https://github.com/C2SM-RCM/c2sm-rcm-env'
+description = 'c2sm environment'
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+sources = []
+source_urls = []
+keepsymlinks = True
+
+commit = 'fea87169a8333504f8e7a85eb2bddbb9be0a9c4a'
+
+postinstallcmds = [
+    ("git clone https://github.com/C2SM-RCM/c2sm-rcm-env.git ."),
+    ("git checkout %s" % commit),
+    ("cp -r modules/kesch/c2sm %(installdir)s")]
```

Comment:
The way it is written,  there is an additional module file that EB automatically generates...
```
easybuild/c2sm-15.08.18-easybuild-devel
```

I think we should add another layer, to avoid this folder. Something like
```
postinstallcmds = [                                                                
    ("git clone https://github.com/C2SM-RCM/c2sm-rcm-env.git ."),                  
    ("git checkout %s" % commit),                                                  
    ("mkdir -p %(installdir)s/kesch"),                                             
    ("cp -r modules/kesch/c2sm %(installdir)s/kesch/"),                            
]                                                                                  
                                                                                   
modtclfooter = """                                                                 
conflict c2sm                                                                      
module use %(installdir)s/kesch                                                    
"""                                                                                
                                                                                   
sanity_check_paths = {                                                             
    'files': ['kesch/c2sm/base', 'kesch/c2sm/gnu_for_cpp'],                        
    'dirs':  ['kesch/c2sm/cray-env', 'kesch/c2sm/gnu-env', 'kesch/c2sm/pgi-env'],
}  
```

What do you think? Does it make sense to you?